docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -157,56 +174,62 @@ CUDA Policies
 * ``cuda_thread_z_direct`` - Direct mapping of loop iterations to cuda threads in the z dimension.
   
 .. note::  
-          * If multiple thread direct policies are used within kernel; the product of the sizes must be :math:`\leq` 1024. 
-          * Repeating thread direct policies with the same thread dimension in perfectly nested loops is not supported. 
-          * Thread direct policies are only recommended with certain loop patterns such as tiling.
+    * Repeating thread direct policies with the same thread dimension in perfectly nested loops is not recommended. Your code may do something, but likely will not do what you expect and/or be correct. 
+    * If multiple thread direct policies are used in a kernel (using different thread dimensions), the product of sizes of the corresponding iteration spaces must be :math:`\leq` 1024. You cannot launch a CUDA kernel with more than 1024 threads per block.
+    * **Thread direct policies are only recommended with certain loop patterns, such as tiling.**
 
-* ``cuda_thread_x_loop`` - Extension to the thread direct policy, introduces a block stride loop based on the thread-block size in the x dimension.
-* ``cuda_thread_y_loop`` - Extension to the thread direct policy, introduces a block stride loop based on the thread-block size in the y dimension.
-* ``cuda_thread_z_loop`` - Extension to the thread direct policy, introduces a block stride loop based on the thread-block size in the z dimension.
+* ``cuda_thread_x_loop`` - Extension to the thread direct policy by introducing a block stride loop based on the thread-block size in the x dimension.
+* ``cuda_thread_y_loop`` - Extension to the thread direct policy by introducing a block stride loop based on the thread-block size in the y dimension.
+* ``cuda_thread_z_loop`` - Extension to the thread direct policy by introducing a block stride loop based on the thread-block size in the z dimension.
 
 .. note::
-          * These polices gives the flexability to have a larger number of iterates than threads in the x/y/z dimension.
-          * There is no constraint on the product of sizes of the associated loop iteration space.
-          * Cuda thread loop policies are recommended for most loop structures.
+    * There is no constraint on the product of sizes of the associated loop iteration space.
+    * These polices enable a having a larger number of iterates than threads in the x/y/z thread dimension.
```

Comment:
Should be "enable having" or "enable a larger number".

Reply:
Reworded so it's clearer.